### ISSUE TITLE
[Legitimate Site Blocked] foundation.app

### ISSUE THREAD

**Issue by jacklynlaryea** [MetaMask/eth-phishing-detect#8617]
https://foundation.app/@meowgress/starseed/5

foundation is a legitimate NFT art website. not sure why they are being blocked by metamask...

**Comment by 409H** [comment]
Closing. Duplicate of https://github.com/MetaMask/eth-phishing-detect/issues/8588
